### ISSUE TITLE
[TW-546] 'verbose no' no longer stops the printing of the 'Project ... complete' message

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#581]
_Peter De Poorter on 2012-04-28T12:26:18Z says:_ 

 I migrated to task 2.0 (ubuntu 12.04)

As you perhaps remember I didn't like the 'Project ... complete' message. In the previous task version I avoided this message by putting verbose on no. In task 2.0 this no longer works. How can I avoid this message ?   

task x done
==> Completed 1 task.
==> Project 'home' is 85% complete (13 of 87 tasks remaining).

Other strange thing: 85% versus 13 of 87. This is clearly not 85%

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-04-28T12:26:18Z
Modified: 2014-02-09T02:04:41Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-04-28T12:38:27Z says:_ 

 Hi Peter, that's not a bug - verbosity controls changed.  See http://taskwarrior.org/projects/taskwarrior/wiki/Feature_verbosity


>> Project 'home' is 85% complete (13 of 87 tasks remaining).
> Other strange thing: 85% versus 13 of 87. This is clearly not 85%

That's 13 of 87 *remaining*.  Therefore 74 of 87 complete.  That's 85%.

**Comment by taskwarrior** [comment]
_Peter De Poorter on 2012-04-28T17:24:37Z says:_ 

 Thanks, Paul. It was indeed not a bug.
Sorry, I was too fast and only looking to the numbers and not the word remaining

**Comment by taskwarrior** [comment]
_Louis-Claude Canon on 2012-04-30T07:03:02Z says:_ 

 The man page, however, may be misleading on this feature. From what I understand from the code, the default behaviour with any setting (including 'no' and 'off') is to show everything but the header (by the way, isn't it a bug? I did not check whether it indeed emulates rc.verbose=off in version 1.9.4).

For clarification in the documentation, I propose the following patch where 'off' is replaced by 'nothing'.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-04-30T12:01:20Z says:_ 

 Thank you Peter and Louis-Claude for pointing all this out.  You are both right.

There are two problems here:
1.  The Documentation does not mention the 'nothing' value
1.  The code does not correctly implement 'verbose=off'

Peter's original complaint is correct - the 'project status' output is no longer controlled by 'verbose=off', and it should be.  There are a couple of other elements also that should be controlled in this way.  Louis-Claude's suspicion that it was wrong is correct.

As a result of the above, the documentation is also wrong, but not quite in the way that Louis-Claude saw it, so I cannot take the patch.  I have made the equivalent changes and provided examples of what 'on', 'off' and 'nothing' mean.

Code and documentation fixed, issue #996 is no longer rejected - it is completed.  Sorry for not seeing the real problem earlier.

**Comment by pbeckingham** [comment]
[TW-546_0008-Documentation.patch.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1756167/TW-546_0008-Documentation.patch.txt)
